incubator/keycloak-proxy/values.yaml

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
If you remove this empty line, CI will probably succeed.

Reply:
@unguiculus Thank you. The test was passed.